tools/params_processors/link_params_processor.py

```diff
@@ -117,15 +117,19 @@ def _process_linkopt(opt):
         if opt.endswith(".o"):
             return None
 
+        # Xcode adds system library search paths
+        if opt.startswith("-L__BAZEL_XCODE_"):
+            return None
+
```

Comment:
@keith There's no harm if these are set twice, right? I should probably just not filter it in case someone does something additional.

Reply:
theoretically maybe the order matters in the case that multiple directories have the overlapping frameworks, but realistically shouldn't happen much

Reply:
https://github.com/buildbuddy-io/rules_xcodeproj/pull/1572